pkg/api/request.go

```diff
@@ -180,8 +183,44 @@ func (a *API) UserGetRequest(w http.ResponseWriter, r *http.Request, requestId s
 		apio.Error(ctx, w, errors.New("access rule result was nil"))
 		return
 	}
+	var options []cache.ProviderOption
```

Comment:
I need to look at factoring this out into teh service as it is repeated a few times, 

I will also look at whether I can reduce it to a single DDB query for all the option for a provider rather than individual options